frechet_audio_distance/fad.py

```diff
@@ -32,7 +34,8 @@ def load_audio_task(fname, sample_rate, dtype="float32"):
         wav_data = wav_data / float(2**31)
 
     # Convert to mono
-    if len(wav_data.shape) > 1:
+    assert channels in [1, 2], "channels must be 1 or 2"

```

Comment:
`channels` here is unspecified, I guess you miss a line before this to calculate`channels` from `wav_data`

Reply:
Yes, it's fixed now. It's an input to the function, defaulting to 1 unless specified (as should be done for encodec 48khz)